flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1,4 +1,10 @@
 {
+    "io.github.nikelaz.CTLDash": {
+        "finish-args-systemd1-talk-name": "Needed to list, monitor, and control user systemd services (start, stop, restart operations on user services).",
+        "finish-args-systemd1-system-talk-name": "Needed to communicate with systemd system daemon via D-Bus to list, monitor, and control system-wide systemd services.",
+        "finish-args-arbitrary-dbus-access": "Need both system and session D-Bus access to manage both system-level and user-level systemd services through the org.freedesktop.systemd1 interface.",
```

Comment:
this is not what you think it is. The very short version unless you have an actual justification to have blank dbus access (like this is a dbus introspector tool) it will not be granted.